Makefile

```diff
@@ -213,129 +251,96 @@ ifneq ($(origin CATALOG_BASE_IMG), undefined)
-OPERATOR_SDK = $(shell pwd)/bin/operator-sdk-$(OPERATOR_SDK_RELEASE)
-OPERATOR_SDK_DL_URL = https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_RELEASE)/operator-sdk_$(OS)_$(ARCH)
-operator-sdk:
-	@if [ ! -f $(OPERATOR_SDK) ]; then\
-		mkdir -p $(shell pwd)/bin;\
-		curl -sL -o $(OPERATOR_SDK) $(OPERATOR_SDK_DL_URL);\
-		chmod +x $(OPERATOR_SDK);\
-	fi
-
-# Download kind locally if necessary
-KIND_RELEASE = v0.11.1
-KIND = $(shell pwd)/bin/kind-$(KIND_RELEASE)
-KIND_DL_URL = https://github.com/kubernetes-sigs/kind/releases/download/$(KIND_RELEASE)/kind-$(OS)-$(ARCH)
-$(KIND):
-	mkdir -p $(shell pwd)/bin
-	curl -sL -o $(KIND) $(KIND_DL_URL)
-	chmod +x $(KIND)
+catalog-image: ## Outputs the catalog image name.
+	@$(MAKE) container-image IMG=$(CATALOG_IMG)
 
-# Download kuttl locally if necessary for e2e tests
-KUTTL_RELEASE = 0.9.0
-KUTTL = $(shell pwd)/bin/kuttl-v$(KUTTL_RELEASE)
-KUTTL_DL_URL = https://github.com/kudobuilder/kuttl/releases/download/v$(KUTTL_RELEASE)/kubectl-kuttl_$(KUTTL_RELEASE)_$(OS)_x86_64
-$(KUTTL):
-	mkdir -p $(shell pwd)/bin
-	curl -sL -o $(KUTTL) $(KUTTL_DL_URL)
-	chmod +x $(KUTTL)
+catalog-push-latest: ## Push the catalog with the `latest` image tag.
+	$(MAKE) container-push \
+			IMG=$(CATALOG_BASE_IMG) CONTAINER_FILE=$(CATALOG_CONTAINER_FILE) CONTAINER_CTX=$(CATALOG_CONTAINER_CTX)
 
 ####################################################
-#### Custom Targets to publish release catalog #####
+##### Custom Targets to release a new version ######
 ####################################################
-##@ Release Catalog
+##@ Release
 
-prepare-release:
+get-new-release:
+	@hack/new-release.sh v$(VERSION)
+
+prepare-release: ## Prepare bundle release files.
 	if echo $(VERSION) | grep -q 'alpha'; \
 		then $(MAKE) prepare-alpha-release; \
 		else $(MAKE) prepare-stable-release; \
 	fi
 
-prepare-alpha-release: bundle ## Prepare alpha release
+prepare-alpha-release: bundle # Prepare alpha release.
 
-prepare-stable-release: bundle ## Prepare stable release
+prepare-stable-release: bundle # Prepare stable release.
 	$(MAKE) bundle CHANNELS=alpha,stable DEFAULT_CHANNEL=alpha
 
-catalog-retag-latest:
-	$(MAKE) container-push \
-			IMG=$(CATALOG_BASE_IMG) CONTAINER_FILE=$(CATALOG_CONTAINER_FILE) CONTAINER_CTX=$(CATALOG_CONTAINER_CTX)
-
-bundle-publish: prepare-release bundle-push ## Publish new bundle
-
-catalog-publish: catalog-add-bundle catalog-push catalog-retag-latest ## Builds and pushes the catalog image
+bundle-publish: prepare-release bundle-push ## Publish new bundle.
 
-release-publish: container-push bundle-publish catalog-publish ## Publish a new stable release (operator, catalog and bundle)
+catalog-publish: catalog-add-bundle catalog-push catalog-push-latest ## Build and push the catalog image.
 
-get-new-release:
-	@hack/new-release.sh v$(VERSION)
+release-publish: container-push bundle-publish catalog-publish ## Publish a new stable release (operator, catalog and bundle).
```

Comment:
Accordign to alpha release docs https://github.com/3scale-ops/prometheus-exporter-operator/blob/main/docs/release.md#alpha

This target need to be executed manually for alpha releases, however the target help comment says it is only for new "stable" releases

I guess this target is shared for both alpha and stable right? so target help comment should be changed?

Reply:
```suggestion
release-publish: container-push bundle-publish catalog-publish ## Publish a new release (operator, catalog and bundle).
```